### PR TITLE
MCR-2114 create pi command

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2114).

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 45f11c84ee0eb03f10a09ae63c724e63c0d3a7d5! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/4990-pull_request-868

**Comment by kkrebs** [comment]
If this should be part of 2019.06, than base have to be changed.

**Review by kkrebs — APPROVED** [review]

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/mycore/mycore/pullRequest?prid=4254170&bid=14529665)

File: mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java [hunk 1]
```diff
@@ -202,4 +203,38 @@ public static void updateObject(String objectIDString) {
         MCRPersistentIdentifierEventHandler.updateObject(object);
     }
 
+    @MCRCommand(syntax = "create pi with {0} for object {1} with additional ({2})",
```
Comment (adlerre) [inline mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java]: Codacy Issue found: [Line is longer than 120 characters (found 137).](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4254170)

File: mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java [hunk 2]
```diff
@@ -202,4 +203,38 @@ public static void updateObject(String objectIDString) {
...
+            .getInstance().getRegistrationService(serviceID);
+
+        if (registrationService.isCreated(objectID, "")) {
+            LOGGER.info("Object {} already has a DOI!", objectID);
+        }
+        final MCRBase object = MCRMetadataManager.retrieve(objectID);
+        final MCRPersistentIdentifier doi = registrationService.register(object, additional);
+        LOGGER.info("Registered pi with {}: {} for object {}", serviceID, doi.asString(), objectID);
+    }
+
```
Comment (adlerre) [inline mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java]: Codacy Issue found: [Line is longer than 120 characters (found 135).](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4254170)

File: mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java [hunk 3]
```diff
@@ -202,4 +203,38 @@ public static void updateObject(String objectIDString) {
         MCRPersistentIdentifierEventHandler.updateObject(object);
     }
 
+    @MCRCommand(syntax = "create pi with {0} for object {1} with additional ({2})",
+        help = "Creates a persistent identifier with the pi service with the id {0} for the object {1}"
```
Comment (adlerre) [inline mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java]: Codacy Issue found: [Line is longer than 120 characters (found 156).](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4254170)

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/mycore/mycore/pullRequest?prid=4254170&bid=14529665)

File: mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java (hunk 1 above)
Comment (adlerre) [inline mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java:206]: Codacy Issue found: [Line is longer than 120 characters (found 137).](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4254170)

File: mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java (hunk 2 above)
Comment (adlerre) [inline mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java:230]: Codacy Issue found: [Line is longer than 120 characters (found 135).](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4254170)

File: mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java (hunk 3 above)
Comment (adlerre) [inline mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java:207]: Codacy Issue found: [Line is longer than 120 characters (found 156).](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4254170)

**Pull request merged**